src/main/common/draft/draftMiddleware.ts

```diff
@@ -1,40 +1,77 @@
 import * as express from 'express'
 import DraftStoreClient from 'common/draft/draftStoreClient'
+import { Draft, DraftDocument } from 'models/draft'
+import { DraftStoreClientFactory } from 'common/draft/draftStoreClientFactory'
 
-export class DraftMiddleware<T> {
-  client: DraftStoreClient<T>
+/**
+ * Extracts external ID from paths in the '/case/<external-id>.*' format
+ */
+function extractExternalId (path: string): string {
+  const pattern = /\/case\/([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})\/.*/
 
-  constructor (public draftType: string, public deserializeFn: (value: any) => T = (value) => value) {
-    if (!draftType || draftType.trim() === '') {
-      throw new Error('Draft type is required to instantiate middleware')
-    }
-    this.client = new DraftStoreClient<T>(draftType)
+  const externalIds: RegExpMatchArray = path.match(pattern)
+  if (externalIds !== null) {
+    return externalIds[1]
   }
+  return undefined
```

Comment:
not exception?

Reply:
I left this undefined as this method might not return UUID in case URL does not have it. Will add this detail to the JSDoc